### DISCUSSION TITLE
quay.io/rook/ceph - can't pull

### DISCUSSION THREAD

**Discussion by steveyears** [rook/rook#14584] [Q&A]
I'm trying to build a rook-ceph cluster on microk8s using the built in helm chart and this cluster yaml:

```
spec:
  cephVersion:
    image: quay.io/rook/ceph:v17.2.7
```
    
and I keep getting:

Failed to pull image "quay.io/rook/ceph:v17.2.7": rpc error: code = NotFound desc = failed to pull and unpack image "quay.io/rook/ceph:v17.2.7": failed to resolve reference "quay.io/rook/ceph:v17.2.7": quay.io/rook/ceph:v17.2.7: not found 

Quay.io seems to be having problems at the moment with Security Scanning, but not sure how this would impact pulling an image. 

When I went to https://quay.io/repository/rook/ceph the page is EMTPY?

**Comment by travisn** [comment]
The Rook repo is on dockerhub:
https://hub.docker.com/r/rook/ceph/tags/
